src/app/profile/yourGiving/historicalView/historicalGift/historicalGift.tpl.html

```diff
@@ -17,7 +17,7 @@ <h4 class="mb0">{{$ctrl.gift['historical-donation-line']['designation-name']}}</
                 <li>
                   <h4>
                     {{$ctrl.gift['historical-donation-line']['amount'] | currency:'$'}}
-                    <span ng-if="$ctrl.gift['recurringdonationelement']"> &mdash; {{$ctrl.gift['recurringdonationelement']['rate']['recurrence']['interval']}}</span>
+                    <span ng-if="$ctrl.gift['recurringdonations']['donations'].length"> &mdash; {{$ctrl.gift['recurringdonations']['donations'][0]['rate']['recurrence']['interval']}}</span>
```

Comment:
Right now, only show the first one. Should it loop over them and show each? I hadn't encountered any historic donations with multiple recurring in the development nor did any UI I've seen show it, so didn't know it was possible.